app/com/m3/octoparts/aggregator/service/PartResponseLocalContentSupport.scala

```diff
@@ -0,0 +1,29 @@
+package com.m3.octoparts.aggregator.service
+
+import com.m3.octoparts.aggregator.PartRequestInfo
+import com.m3.octoparts.model.PartResponse
+import com.m3.octoparts.model.config.{ HttpPartConfig, ShortPartParam }
+
+import scala.concurrent.Future
+
+trait PartResponseLocalContentSupport extends PartRequestServiceBase {
+
+  override def processWithConfig(ci: HttpPartConfig,
+                                 partRequestInfo: PartRequestInfo,
+                                 params: Map[ShortPartParam, Seq[String]]): Future[PartResponse] = {
+    if (ci.localContentsEnabled) {
```

Comment:
This pull request's title links to #76. But I think this pull request just suggests new feature to use static content prepared inside Octoparts instead. Following use cases can be considered.

- use dummy content instead when working on integration with backends that are not implemented yet
- manually choose static content to show 'sorry page' instead when backends are in trouble

If Octoparts automatically returns alternative content when the part is in trouble, that's pretty cool. But the feature can be separated. At least @reki2000 and I suggested so because this simple feature can solve our current issues (easy stubs for backend, stable CI builds) on some level. 

Of course, it's also fine if we can choose the `local content` as fallback content. What do you think about?